main.md

```diff
@@ -288,7 +283,30 @@ In the context of a request object as specified in [@I-D.ietf-oauth-jwsreq], `au
 
 Note: Authorization request URIs containing authorization details in a request parameter or a request object can become very long. Implementers SHOULD therefore consider using the `request_uri` parameter as defined in [@I-D.ietf-oauth-jwsreq], potentially in combination with the pushed request object mechanism as defined in [@I-D.lodderstedt-oauth-par] to pass authorization details in a reliable and secure manner.
 
-## Authorization Request Processing 
+Here is an example of a pushed authorization request that sends the authorization request data directly to the AS via a HTTPS-protected connection: 
```

Comment:
Add an informative reference to the PAR draft here.

Reply:
There is a PAR ref in the previous paragraph. You think it's needed here too?

Reply:
I merged both paragraphs to make the single PAR reference unambiguous.